index.html

```diff
@@ -1,114 +1,223 @@
-      </p>
-
-      <h2>Report abuse</h2>
-      <p>
-        If you believe one of the subdomains is abusing the service, please <a href="https://github.com/is-a-dev/register/issues/new?labels=report-abuse&template=report-abuse.md&title=Report+abuse" target="_blank">report</a> it.
-      </p>
-
-      <h2>Donations</h2>
-      <p>This project is a free service for developers and will stay that way. Please consider <a href="https://www.buymeacoffee.com/phenax">donating</a> to help me keep this running forever!</p>
-
-      <p>
-        <a href="https://www.buymeacoffee.com/phenax" target="_blank"><img src="https://cdn.buymeacoffee.com/buttons/default-orange.png" alt="Buy Me A Coffee" height="28" width="119"></a>
-        <a href="https://liberapay.com/phenax" target="_blank"><img src="https://img.shields.io/badge/liberapay-donate-yellow.svg?style=for-the-badge" alt="Liberapay recurring donation button" /></a>
-      </p>
+        <div class="socials">
+            <div class="social">
+                <a href="https://github.com/is-a-dev" target="_blank"><i class="fa-brands fa-github"></i></a>
+            </div>
+        </div>
+        <h1>is-a.dev<p>Welcome! 👋</p>
+        </h1>
+        <p>
+            <span>is-a.dev</span> is a service that allows developers to get a sweet-looking <span
+                style="display:inline-block;width:fit-content;line-height:1;">.is-a.dev</span> subdomain for their
+            personal
+            websites.
+            <br />
+            To register your own subdomain, go to the <a href="https://github.com/is-a-dev/register"
+                target="_blank">is-a.dev
+                registration repository</a> and follow the instructions.
+        </p>
+
+        <h2>Report abuse</h2>
+        <p>
+            If you believe one of the subdomains is abusing the service, please <a
+                href="https://github.com/is-a-dev/register/issues/new?labels=report-abuse&template=report-abuse.md&title=Report+abuse"
+                target="_blank">report</a> it.
+        </p>
+
+        <h2>Donations</h2>
+        <p>This project is a free service for developers and will stay that way. Please consider <a
+                href="https://www.buymeacoffee.com/phenax">donating</a> to help me keep this running forever!</p>
+
+        <p>
+            <a href="https://www.buymeacoffee.com/phenax" target="_blank"><img
+                    src="https://cdn.buymeacoffee.com/buttons/default-orange.png" alt="Buy Me A Coffee" height="28"
+                    width="119"></a>
+            <a href="https://liberapay.com/phenax" target="_blank"><img
+                    src="https://img.shields.io/badge/liberapay-donate-yellow.svg?style=for-the-badge"
+                    alt="Liberapay recurring donation button" /></a>
+        </p>
     </main>
-  </body>
+</body>
+
```

Comment:
```suggestion
```

Reply:
It is licensing, generally you'd want to leave that in but ig I can remove it?

Reply:
@TheRedXD Half of the licenses are just blank and also they aren't referenced anywhere either. We can always add attribution in the footer.